config/initializers/apipie.rb

```diff
@@ -27,6 +27,11 @@
     trans
   end
   config.help_layout = 'apipie_dsl/apipie_dsls/help.html.erb'
+  config.default_class_description = ->(model) do
```

Comment:
I think RuboCop wants to use lambda:
```suggestion
  config.default_class_description = lambda do |model|
```